test/specs/validate-spec/invalid/3.0/component-schema-with-invalid-characters.yaml

```diff
@@ -0,0 +1,25 @@
+openapi: 3.0.3
+info:
+  version: "1.0"
+  title: Invalid API
+paths:
+  /anything:
+    get:
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  "$ref": "#/components/schemas/User«Information»"
+components:
+  schemas:
+    User«Information»:    # <---- component names should match ^[a-zA-Z0-9\.\-_]+$
```

Comment:
Per the spec: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#components-object

> All the fixed fields declared above are objects that MUST use keys that match the regular expression: ^[a-zA-Z0-9\.\-_]+$.